cmd/starter/main.go

```
@@ -218,3 +237,23 @@ func uploadAssets(repo, tmpDir string) {
 		orFail(err)
 	}
 }
+
+func triggerSubBuilds(sourcerepo string, repos []string) {
+	if len(repos) > 20 {
+		// Flood / DDoS protection
+		log.WithFields(logrus.Fields{
+			"repository":         sourcerepo,
+			"number_of_triggers": len(repos),
+		}).Error("Too many triggers passed")
+		return
+	}
+
+	for _, repo := range repos {
+		go func(repo string) {
+			resp, _ := http.PostForm("https://gobuilder.me/build", url.Values{
```

ignored error... log it at least?